server/routes/volunteers.ts

```diff
@@ -128,6 +130,176 @@ router.route("/api/volunteers/").get(
   })
 );
 
+/**
+ * @description Update a volunteers info
+ * @route  PATCH /api/volunteers/update/:volunteerId
+ * @access
+ */
+router.route("/api/volunteers/update/:volunteerId").patch(
+  protect,
+  asyncHandler(async (req: Request, res: Response) => {
+    const { volunteerId } = req.params;
+    logger.info(`PATCH /api/volunteers/update/${volunteerId}`);
+    logger.info("Request body: ", req.body);
+
+    const {
+      firstName,
+      lastName,
+      email,
+      phoneNumber,
+      participantType,
+    }: {
+      firstName: string;
+      lastName: string;
+      email: string;
+      phoneNumber: string;
+      participantType: string[];
+    } = req.body;
+
+    const stringFields = [firstName, lastName, email, phoneNumber];
+
+    const isValidRequest = stringFields.every((field) => {
+      return field.trim().length > 0;
+    });
+
+    // also verify that the participantType is valid
+    const validParticipantTypes = ["Driver", "Packer"];
```

Comment:
Doesn't this need to include "Driver & Packer", too?

Reply:
Overall, this is a really impressive PR, @6mp!  Would be great if some others could review the code (@jasoncavanaugh,  @zuechai). 
Also, @danthefridgeman, just checking on the functionality here you want. With this little popup, the admin will be able to edit the person's actual name, phone number, and email in the master CRM table. That's pretty cool, but also a lot of power so want to make sure that's desired in this spot.

Reply:
> Doesn't this need to include "Driver & Packer", too?

If a volunteer is both a driver and packet I send and array with `["Driver", "Packer"]` since thats the format airtable takes it in. If they're one or the other it's just a single array like `["Driver"]`.

As for permissions I will update this endpoint to only be used by admins afterr I make my pr for issue #52 which will   restrict functionally of the admin portal to specified users.

Reply:
Really great job! Curious to see the solution to the time slot discussion.